src/components/InRequest/InRequestViewCompact.tsx

```diff
@@ -0,0 +1,126 @@
+import { FunctionComponent, useEffect, useState } from "react";
+import { EMPTYPES } from "../../constants/EmpTypes";
+import { makeStyles, Label, Text } from "@fluentui/react-components";
+import { InRequestEditPanel } from "./InRequestEditPanel";
+import { IInForm } from "../../api/RequestApi";
+
+/* FluentUI Styling */
+const useStyles = makeStyles({
+  compactContainer: {
+    display: "grid",
+    paddingLeft: "1em",
+    paddingRight: "1em",
+    gridTemplateColumns: "repeat(auto-fit, minmax(150px,1fr))",
+    gridAutoRows: "minmax(50px, auto)",
+  },
+});
+
+export interface IInRequestViewCompact {
+  formData: IInForm;
+}
+
+export const InRequestViewCompact: FunctionComponent<IInRequestViewCompact> = (
+  props
+) => {
+  const classes = useStyles();
+  const [formData, setFormData] = useState<IInForm>(props.formData);
+
+  // Watch for when the data is loaded in the parent, and pass down to this component
+  useEffect(() => {
+    setFormData(props.formData);
+  }, [props.formData]);
```

Comment:
Maybe you can help me here.  I had tried using just the props earlier, but found that when I pushed an update to that prop from the EditPanel it didn't refresh the content.  Only by making it a state did it refresh correctly.

Reply:
After reading the article you sent, it clicked.  Props don't force a re-render, state changes do.  If I move the component up a level it then becomes a child of the component I am updating the state of (instead of a sibling).  So the changed state at the  parent will then force the children to re-render too -- making the changed props be applied.